_posts/2019-12-06-cfep09.md

```diff
@@ -1,3 +1,10 @@
+---
+layout: post
+title: conda-forge CFEP-09
```

Comment:
@CJ-Wright this is your blog post. Do you want to change the title?

Reply:
Most likely, although not certain to what yet.